platform/viewer/src/appExtensions/MeasurementsPanel/index.js

```diff
@@ -7,10 +8,42 @@ export default {
    */
   id: 'measurements-table',
 
-  preRegistration({ servicesManager, configuration = {} }) {
-    init({ servicesManager, configuration });
+  preRegistration({
+    servicesManager = {},
+    commandsManager = {},
```

Comment:
Same here. If `servicesManager` and `commandsManager` are undefined here, I want everything to explode.

Reply:
Updated.